### PR TITLE
Add rubocop extensions

### PR DESCRIPTION
**Pull request by benmelz**
Now I'm not 100% on `rubocop-rails` - it does sound good to have the normal ActiveRecord cops included (it even caught a thing or two), but I'm not totally sure it's warranted

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+  - rubocop-rails
+  - rubocop-rake
+  - rubocop-rspec
 
 AllCops:
-  Include:
-    - Capfile
-    - Gemfile
-    - Rakefile
-    - config.ru
```
Comment (benmelz) [inline .rubocop.yml:8]: All of these get automatically picked up

File: Rakefile [hunk 2]
```diff
@@ -7,12 +7,4 @@ Bundler.require(:default, env)
...
 
 # development and production use an unmodifiable db
-if env == 'test'
-  require 'sinatra/activerecord/rake'
-
-  namespace :db do
-    task :load_config do
-      require 'eam_app'
-    end
-  end
```
Comment (benmelz) [inline Rakefile:17]: I couldn't really figure out what this is for - everything seems to work okay without it

Comment (werebus) [reply Rakefile:17]: :shrug:

**Review by werebus — APPROVED** [review]
> ...I'm not 100% on rubocop-rails...

Eh, why not? As long as we don't find ourselves disabling too much, I'd say we have just as much "rails" code in this app as, say, "factory bot" code.

**Pull request merged**